debian/Makefile

```diff
@@ -97,9 +147,22 @@ else
 	RPM_RELEASE_ID=$(REVISION)
 endif
 
-rpm: RPM_BUILDING/SOURCES/$(FULL_PACKAGE_TITLE)-$(RPM_VERSION).tar.gz
+rpm: rpm-amd64 clean-build-files
+	make -f debian/Makefile rpm-arm64
```

Comment:
So this is kind of silly, but the reason I'm running the second build in a sub-shell is to get make to actually run targets like `rpm-build-area` to run again, which they otherwise won't because they're already called by `rpm-amd64`.

We can clean this up eventually.